other/iris/iris.jl

```diff
@@ -37,8 +37,8 @@ end
 
 
 function get_processed_data(args)
-    labels = Flux.Data.Iris.labels()
-    features = Flux.Data.Iris.features()
+    labels = Flux.load_iris()[2]
+    features = Flux.load_iris()[1]
```

Comment:
Datasets are now just an `] add` away in MLDatasets.jl

Reply:
Understand the point, I am sorry for not using correct version of Flux, which still contains the data link.

Related to the above and below, what is the best way to do the edit? 

1) Update Flux, add MLDatasets, use the most updated versions and adjust the code?
2) Or just keep the one edit below, adding.`, params`